modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarPage.testcase

```diff
@@ -155,11 +155,11 @@ definition {
 
 		IFrame.closeFrame();
 
-		CalendarEvent.disableEventBySiteCalendarName(calendarName = "Liferay DXP");
+		CalendarEvent.disableEventBySiteCalendarName(calendarName = ${siteName});
```

Comment:
Hey @patriciajeanperez, It took me a while to understand what you explained to me. I'm sorry.
It's done. Thanks for your guidance.